utils/src/types/items.types.ts

```diff
@@ -118,4 +118,21 @@ export const ITEMS: Record<ItemName, Item> = {
     imageUrl: IMAGE_HEADER + 'fishing.jpg',
     ranking: 7,
   },
-};
\ No newline at end of file
+};
+
+/**
+ * Get Lost at Sea item item pair ranking answer.
+ */
+export function getLostAtSeaPairAnswer(item1: ItemName, item2: ItemName) {
```

Comment:
Nice, we can use this logic for multiple choice too (1 item has ranking 1, others 0).

Perhaps we can generalize the naming since this doesn't just apply to LostAtSea

Reply:
Hmm right now the array used is specific to LAS, but I like the idea of generalizing it later. Maybe we can add a note to refactor it at that point?

Reply:
Sounds good, yes!